packages/bcal/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/jarun/bcal
+TERMUX_PKG_DESCRIPTION="Command-line utility for storage conversions and calculations"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@fervi"
+TERMUX_PKG_VERSION="2.4"
+TERMUX_PKG_SRCURL=https://github.com/jarun/bcal/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=141f39d866f62274b2262164baaac6202f60749862c84c2e6ed231f6d03ee8df
+TERMUX_PKG_DEPENDS="bc, readline"
+TERMUX_PKG_BLACKLISTED_ARCHES="arm, i686"
+
+termux_step_pre_configure() {
+	make install
```

Comment:
Why is this needed?

Reply:
Architecture Blacklist? This application is only 64-bit
https://packages.debian.org/sid/bcal

Reply:
make install, cause build script return:
> ERROR: No files in package.

Reply:
I think that error message needs to be improved
You can use `TERMUX_PKG_BUILD_IN_SRC=true`

It will be better if you add a link or comment on why the blacklisted arches too

Reply:
@truboxl Fixed, thanks